projects/npm-tools/packages/npm-scripts/src/utils/getMergedConfig.js

```diff
@@ -123,12 +147,20 @@ function getMergedConfig(type, property) {
 			}
 			break;
 
-		case 'bundler':
-			mergedConfig = deepMerge([
-				require('../config/npm-bundler'),
-				getUserConfig('npmbundler'),
-			]);
+		case 'bundler': {
+			const {bundler: bundlerDefaults} = getMergedConfig(
+				'npmscripts',
+				'build'
```

Comment:
Just checking: so this will `throw` if there is no `build` key; and that's ok to enforce because we provide one by default?

Reply:
I think it is impossible that you arrive to this point without having a `build` config. Otherwise the bundler wouldn't be invoked.

However, I'll rewrite it just in case, because it may be assuming too much...